.travis.yml

```diff
@@ -31,7 +31,11 @@ addons:
 
 # Build your code e.g. by calling make
 script:
-  - doxygen Doxyfile
+  - mkdir build
+  - cd build
+  - cmake ..
+  - make kilosim_docs
+  - make install
```

Comment:
I think the Travis failure is coming from here -- same issue as I mentioned elsewhere. `make install` fails because it's trying to find `build/kilosim_example`, which hasn't been generated.